Wrapping/Python/Examples/use_case_02.py

```diff
@@ -11,7 +11,16 @@
 import dream3d.simpl_helpers as simpl_helpers
 
 
-
+#Generates a csv file with floats 0-37988 and ints 1-37989
+def make_random_data():
```

Comment:
Could we add an argument to the function `num_elements` so that callers of the data can set the number of values to be written to the csv file?
Also we might want to add an argument for the output file so that a user of the function can set the output file.